cloud/libvirt/actuators/machine/actuator.go

```diff
@@ -35,6 +37,7 @@ type ActuatorParams struct {
 
 // NewActuator creates a new Actuator
 func NewActuator(params ActuatorParams) (*Actuator, error) {
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
nice, thanks. Can we include a comment to include context for others when seeing this?